tests/debian_test.yaml

```diff
@@ -13,6 +23,18 @@ commandTests:
   command: 'sh'
   args: ['-c', 'echo $PATH']
   expectedOutput: ['/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin']
+- name: "Test Capabilities ContainerRunOptions"
+  command: "capsh"
+  args: ["--print"]
+  expectedOutput:
+    - ".*cap_sys_admin.*"
+- name: "Test bindMounts containerRunOptions"
+  command: "test"
+  args:
+    - "-d"
+    - "/tmp/test"
+  exitCode: 0
```

Comment:
I had to test this using the command module, since it looks like the fileExistenceTests only work on the image content. Hopefully this is all fine.

Reply:
I'm not sure I totally understand, can you explain a little more what you're actually testing here?

Reply:
I had tested this using the fileExistenceTests, but the test was failing. I think that's because the fileExistenceTests don't run inside the running container, but on the underlying image data? (maybe i've misunderstood).

So instead, I use the command module and run `test -d /tmp/test` which i've added as a bindMount. This tests that a file exists at path `/tmp/test` and that it's of type directory.

Reply:
Ah right, so the whole idea of this framework is that *nothing* happens inside the "running container": a new container is created from the base image on each test, as a way to isolate the testing environment. When you make changes through `Setup` command for example, those get made on a running container and then committed to a new image which is used as the base image for all subsequent tests.

The idea here is that you'll want to treat your `runOpts` as a sort of setup, that gets run **before** any of the tests get run, and then committed to a new image. Then you should be able to use the `fileExistence` tests to test this as normal. Does that make sense?

Reply:
Ah ok, I see what you mean.

In that case, I think we'll need to just pass the `runOpts` anytime we create a container in the docker driver (e.g. [here](https://github.com/GoogleCloudPlatform/container-structure-test/blob/master/pkg/drivers/docker_driver.go#L143)). We'll need to just be careful to pass it everywhere since we can't commit this as the "base image".

One idea would be to make the `runOpts` a subset of the [Config](https://godoc.org/github.com/fsouza/go-dockerclient#Config) struct that gets passed around in the client library we're using: then you could just have a method that creates a Config with the values in the `runOpts`, and use that where ever we're called `CreateContainer()`.